Ginger/GingerCoreNET/RunLib/DynamicExecutionLib/DynamicExecutionManager.cs

```diff
@@ -377,13 +378,27 @@ public static void CreateRunSetFromXML(RunsetExecutor runsetExecutor, AddRunset
         #endregion XML
 
         #region JSON
-        public static string CreateDynamicRunSetJSON(Solution solution, RunsetExecutor runsetExecutor, CLIHelper cliHelper)
+        public static string CreateDynamicRunSetJSON(Solution solution, RunsetExecutor runsetExecutor, CLIHelper cliHelper,bool isRemoteExecution=false)
```

Comment:
i prefer no to mess this class related to JSON with the Request, let's think how to separate it but still to reuse